src/components/Divider/divider.module.css

```diff
@@ -0,0 +1,17 @@
+:root {
+  --denhaag-divider-border-color: var(--denhaag-grey-2);
+  --denhaag-divider-border-width: 1px;
+  --denhaag-divider-margin: 16px 16px;
+}
```

Comment:
These should be moved to the basestyles package. Look at the button PR. Creating a file called `tokens-divider.css` and using an `@import "tokens-divider.css"` in `globals.module.css`. It's unlikely that that's the definitive approach, but at the very least it groups our tokens in a single package, making it easier to refactor later.